### PR TITLE
bug: Timeline Spacing #3318

### PR DESCRIPTION
**Pull request by kcherki**
Fix timeline style:
- Added a condition for the "md" size to the `<div class="$$timeline-end">` class to manage spacing on "md" screen sizes: replaced `mb-10` with `md:mb-10`.
- After checking the style, no conflicts were found; it was simply an adjustment for medium screen sizes (md).

**Otherwise, if there are any comments or if this is not the right place, please let me know with your feedback.**

### CODE REVIEW

File: src/components/unstyled/timeline.css [hunk 1]
```diff
@@ -31,5 +31,5 @@
   @apply col-start-2 row-start-2;
 }
 .timeline-end {
-  @apply col-start-1 col-end-4 row-start-3 row-end-4 m-1 self-start justify-self-center;
+  @apply col-start-1 col-end-4 row-start-1 row-end-2 m-1 self-start justify-self-center;
```
Comment (saadeghi) [inline src/components/unstyled/timeline.css:34]: Why?
This is breaking `timeline-end`
Before:
Screenshot 2024-12-21 at 11 13 15 PM

After:
Screenshot 2024-12-21 at 11 13 22 PM

File: src/docs/package-lock.json
Comment (saadeghi) [inline src/docs/package-lock.json:1]: Don't commit lockfiles or formatting please

**Review by saadeghi — CHANGES_REQUESTED** [review]

File: src/components/unstyled/timeline.css (hunk 1 above)
Comment (kcherki) [reply src/components/unstyled/timeline.css:34]: Sorry for the confusion, this change was made only for testing and wasn’t meant to be sent. Thanks for pointing it out!

Comment (saadeghi) [reply src/components/unstyled/timeline.css:34]: Can you explain what's the issue exactly? I can help.

**Pull request closed without merging**